python_modules/dagster/dagster/_core/pipes/subprocess.py

```diff
@@ -20,24 +20,24 @@
 )
 
 
-class _ExtSubprocess(ExtClient):
-    """An ext client that runs a subprocess with the given command and environment.
+class _PipesSubprocess(PipesClient):
+    """A pipes client that runs a subprocess with the given command and environment.
 
     By default parameters are injected via environment variables. And then context is passed via
     a temp file, and structured messages are read from from a temp file.
 
     Args:
         env (Optional[Mapping[str, str]]): An optional dict of environment variables to pass to the subprocess.
         cwd (Optional[str]): Working directory in which to launch the subprocess command.
-        context_injector (Optional[ExtContextInjector]): An context injector to use to inject context into the subprocess. Defaults to ExtTempFileContextInjector.
-        message_reader (Optional[ExtContextInjector]): An context injector to use to read messages from  the subprocess. Defaults to ExtTempFileMessageReader.
+        context_injector (Optional[PipesContextInjector]): An context injector to use to inject context into the subprocess. Defaults to ExtTempFileContextInjector.
+        message_reader (Optional[PipesContextInjector]): An context injector to use to read messages from  the subprocess. Defaults to ExtTempFileMessageReader.
```

Comment:
for message_reader the type in this comment was wrong (and now updated to new wrong)